flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/shuffle/StatisticsOrRecordTypeInformation.java

```diff
@@ -79,11 +78,6 @@ public TypeSerializer<StatisticsOrRecord> createSerializer(SerializerConfig conf
     return new StatisticsOrRecordSerializer(globalStatisticsSerializer, recordSerializer);
   }
 
-  @Override
-  public TypeSerializer<StatisticsOrRecord> createSerializer(ExecutionConfig config) {
```

Comment:
isn't serializer a key part of type information? what's the alternative?

Reply:
The interface doesn't have this method anymore. Note that there is another `createSerializer` method just above (line 76).